disco_aws_automation/spotinst_client.py

```diff
@@ -116,6 +123,11 @@ def _make_request(self, method, path, params=None, data=None):
         :rtype: dict
         """
         try:
+            if params:
```

Comment:
This just looks a touch awkward to me. What do you think about?
```suggestion
            if not params:
                params = {}
            
            params['accountId'] = self.account_id
```